style_guides/pl_sql_packages.md

```diff
@@ -0,0 +1,119 @@
+# Database Packages
+
+## Package Names
+
+The following naming conventions should be used for database packages:
+
+```
+pkw_<name>        -- Package
+pkw_test_<name>   -- UT/PLSQL Test package

```

Comment:
suggest test is added to the end so the files are sorted alphabetically and package files are together

Reply:
good idea.